authors/apps/authentication/serializers.py

```diff
@@ -140,3 +140,58 @@ def update(self, instance, validated_data):
         instance.save()
 
         return instance
+
+
+class ResetEmailSerializer(serializers.ModelSerializer):
+    """
+    Serializer for requesting a password reset e-mail.
+    """
+    email = serializers.EmailField()
+
+    @classmethod
+    def validate_email(self, value):
+        try:
+            valid_email = User.objects.get(email=value)
+        except Exception:
+            raise serializers.ValidationError(f"{value} email is Invalid")
+        return value
+
+    class Meta:
+        model = User
+        fields = ['email']
+
+
+class PasswordResetSerializer(serializers.ModelSerializer):
+    password1 = serializers.CharField(
+        max_length=128,
+        min_length=8,
+        required=True,
+        write_only=True,
+        style={'input_type': 'password', 'placeholder': 'Password'})
+    password2 = serializers.CharField(
+        max_length=128,
+        min_length=8,
+        required=True,
+        write_only=True,
+        style={'input_type': 'password', 'placeholder': 'Password'})
+
+    @classmethod
+    def clean(self):
+        cleaned_data = super(ConfirmPassword, self).clean()
+        password1 = cleaned_data.get('password1')
+        password2 = cleaned_data.get('password2')
+        if password1 != password2:
+            raise serializers.ValidationError('Password does not match.')
+            
+    @classmethod
+    def save(self, commit=True):
+        user = super(ConfirmPassword.password1, self).save(commit)
+        user.last_login = datetime.datetime.now()
+        if commit:
+            user.save()
+        return user
+
+    class Meta:
```

Comment:
@ManuelDominic , there is no need to confirm the password in the backend.
This can be done in the front end